docs/HowTo/Store-Keys/Use-Hashicorp.md

```diff
@@ -54,15 +52,32 @@ option set to `8590` to avoid conflict with the default EthSigner listening port
     protection. That is, the genesis file must include at least the Spurious Dragon milestone
     (defined as `eip158Block` in the genesis file) so the blockchain is using a chain ID.
 
+## Create the Known Servers File
+
+Create a file (in this example, `knownHashicorpServers`) that lists one or more
+trusted Hashicorp Vault servers. The file contents use the format
+`<hostame>:<port> <hex-string>` where:
+
+* `<hostname>` is the server hostname
+* `<port>` is the port used for communication
+* `<hex-string>` is the SHA-256 fingerprint of the server's certificate.
+
+!!! example
+    ```
+    localhost:8200 7C:B3:3E:F9:98:43:5E:62:69:9F:A9:9D:41:14:03:BA:83:24:AC:04:CE:BD:92:49:1B:8D:B2:A4:86:39:4C:BB
+    127.0.0.1:8200 7C:B3:3E:F9:98:43:5E:62:69:9F:A9:9D:41:14:03:BA:83:24:AC:04:CE:BD:92:49:1B:8D:B2:A4:86:39:4C:BB
+    ```
+
 ## Start EthSigner with Hashicorp Vault signing
 
 Start EthSigner.
 
 !!! example
 
     ```bash
-    ethsigner --chain-id=2018 --downstream-http-port=8590 hashicorp-signer --host=127.0.0.1 --port=8200 --auth-file=authFile
+    ethsigner --chain-id=2018 --downstream-http-port=8590 hashicorp-signer --host=127.0.0.1 --port=8200 --auth-file=authFile --tls-known-server-file=/Users/me/my_node/knownHashicorpServers
     ```
+To disable TLS set [`hashicorp-signer --tls-enabled`](../../Reference/CLI/CLI-Syntax.md#tls-enabled) to `false`.
```

Comment:
To disable TLS set -> To disable TLS, set

Reply:
Done